### PR TITLE
bump(github.com/Sirupsen/logrus) 

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@php-coder ptal

File: vendor/github.com/Sirupsen/logrus/alt_exit_test.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
+package logrus
```
Comment (php-coder) [inline vendor/github.com/Sirupsen/logrus/alt_exit_test.go]: Most likely we should exclude test files like this.

File: vendor/github.com/Sirupsen/logrus/logrus_test.go [hunk 2]
```diff
@@ -255,30 +255,58 @@ func TestParseLevel(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, PanicLevel, l)
 
+	l, err = ParseLevel("PANIC")
```
Comment (php-coder) [inline vendor/github.com/Sirupsen/logrus/logrus_test.go]: And this file too.

**Comment by bparees** [comment]
@php-coder test files removed

**Comment by php-coder** [comment]
LGTM.

**Comment by bparees** [comment]
[merge]

**Pull request merged**